.factory/tests/test_fix_issue_pr_body.py

```
@@ -0,0 +1,23 @@
+契约面（脚本即执行计划）：body 行丢失即测试失败。
+"""
+from pathlib import Path
+
+FIX = Path(__file__).resolve().parent.parent / "fix-issue.sh"
+SRC = FIX.read_text(encoding="utf-8")
+
+
+def test_pr_create_carries_closes_body():
+    # 完整契约行（截取 body 起始即可：误删/改前缀均失配）
+    body = '--body-file <(echo "Closes #${ISSUE}"; echo;'
+    assert body in SRC, "pr create 必须带 Closes body（链约定：PR body 含 Closes #N）"
```

Ruff (0.16.3)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)